### PR TITLE
Captstone Project: Metrics webapp

### PR DESCRIPTION
**Pull request by veronica365**
### Description

- Render a list of categories filtered by some category
- Show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,182 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[Live Demo Link](https://countries-data-4ptm.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Walkthrough <a name="walk-through"></a>
+
+-[Walk through](https://www.loom.com/share/9ad69e78c3444b1d9ebfc45ddfdb33ed)
```
Comment (DammyShittu) [inline README.md]: - [ ] In your video presentation you failed to show the actual code (in VS Code for example) you are proud of or find interesting. Kindly remake your video including that part.

File: src/components/CountryList/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          && countries.map((country) => (
+            <CountryItem key={country.id} country={country} />
+          ))}
+      </section>
+    );
+  }
+  return (
+    <section className="countries">
+      {countryById.id && <CountryDetail country={countryById} />}
+    </section>
```
Comment (DammyShittu) [inline src/components/CountryList/index.js]: - [ ] Kindly style the countries on the homepage to follow the design we have in the design. The design has alternating color patterns, but yours does not. Kindly add that.

  Your Implementation | Design guideline
  :---:|:---:
  udjs | uiww

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @veronica365,

## Highlights
✔️ Search feature
✔️ Passing tests

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @veronica365👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**